packages/edit-site/src/components/sidebar-navigation-screen-template/index.js

```diff
@@ -42,6 +43,11 @@ function useTemplateTitleAndDescription( postType, postId ) {
 		);
 	}
 
+	let content = null;
+	if ( record?.slug === 'home' || record?.slug === 'index' ) {
```

Comment:
If this applies to all blog templates, shouldn't it include the Archives template too? Or should it only apply to Home, with Index as a fallback if Home doesn't exist? Settings such as "site title" only really make sense on Home I guess 🤔

Reply:
Good question! I wasn't really sure about this. Happy to take instructions from @jameskoster here 😄

Reply:
No, because Archive and so on can display custom post types too. It may be that we add a local posts-per-page setting to to templates like Archive in the future, but let's start with the global stuff for now.

> Settings such as "site title" only really make sense on Home I guess

The setting here should be Blog title, not Site title. And it should only be visible when a Posts Page is set. Updating the Blog title simply updates the title of the Posts Page.

Reply:
> Updating the Blog title simply updates the title of the Posts Page.

@jameskoster Ah, that was a misinterpretation on my part, sorry!

I've updated this PR so that the "Blog title" input control only shows when a static page is selected as the posts page. The value of the input control updates the title of that page.

<img width="499" alt="Screenshot 2023-06-14 at 2 09 49 pm" src="https://github.com/WordPress/gutenberg/assets/6458278/20bdd7c1-5e87-46f2-86b7-2f1c57a1d97b">

Reply:
Just reflecting on the above change, I think I was unclear about why we would want to update the title of the posts page from the home template, when the two could have no relationship. For example, if I select page `X` as my posts page, it might use a page template and not the home or index templates.

Unless the intention is to surface general site settings?

The only other conclusion I could reach is we'd show these controls when we set a static page as the posts page and _**that**_ static page uses the home/index template. 

Anyway, let me know if I'm way off.

Reply:
> For example, if I select page X as my posts page, it might use a page template and not the home or index templates.

Please feel free to double-check but I think the Posts page will _always_ resolve to either Home or Index. Note that the template switching UI in the post editor does not allow you to change the template when editing this page;

<img width="324" alt="Screenshot 2023-06-14 at 11 05 59" src="https://github.com/WordPress/gutenberg/assets/846565/1ed60576-29f9-4a08-b2db-8b4a3bc421d5">

> I've updated this PR so that the "Blog title" input control only shows when a static page is selected as the posts page. The value of the input control updates the title of that page.

If the above is true, this sounds perfect!

Reply:
> I think the Posts page will always resolve to either Home or Index. 

Oh yes, good one! Another piece of the puzzle falls into place. 😄 

>  I've updated this PR so that the "Blog title" input control only shows when a static page is selected as the posts page. The value of the input control updates the title of that page.

Just to confirm, and maybe you're aware of this already. Here's what I'm seeing:

1. Select a page to act as the posts page (it uses the home page template in 2022 and 2023)
<img width="499" alt="Screenshot 2023-06-14 at 2 09 49 pm" src="https://github.com/WordPress/gutenberg/assets/6458278/20bdd7c1-5e87-46f2-86b7-2f1c57a1d97b">

2. Here's the page in question. The title is highlighted
<img width="800" alt="Screenshot 2023-06-16 at 10 38 04 am" src="https://github.com/WordPress/gutenberg/assets/6458278/5b6af5d2-ac66-4b02-830a-3d4e70e1cd2c">

3. When updating the post page title, the only thing that will change on the home page is the nav menu item.
<img width="800" alt="Screenshot 2023-06-16 at 10 37 56 am" src="https://github.com/WordPress/gutenberg/assets/6458278/e382b1b0-b3a7-401a-9469-e8f20a019818">